custom/icds_reports/tests/agg_tests/test_aggregation_script.py

```diff
@@ -92,6 +92,8 @@ def _load_data_from_db(self, table_name, sort_key, filter_by=None):
     def _load_and_compare_data(self, table_name, path, sort_key=None, filter_by=None):
         # To speed up tests, we use a sort_key wherever possible
         #   to presort before comparing data
+        if 'css_record_monthly' in table_name:
```

Comment:
was this just for testing? Can you remove this if yes.

Reply:
aah yes